Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'jquery-rails'
 # Markdown
 gem 'redcarpet'
 
+gem 'metainspector'
```

Comment:
how about a comment saying what this does, to help future devs doing a gem clean out one day